requirements.txt

```diff
@@ -0,0 +1,42 @@
+# Django and Wagtail
```

Comment:
we use [poetry](https://python-poetry.org/docs/), so this requirements file is extraneous